watch.js

```diff
@@ -137,30 +126,20 @@ module.exports = options => {
         }
         // TODO track dependencies to make it easy to rebuild files as new ones get
         // added, eg logic is added, we need to rebuild upwards
-        if (isData(name)) {
-          return data[name]
-            ? `import ${toCamelCase(name)} from '${getImportFileName(
-                name,
-                file
-              )}'`
-            : dataNotFound(name)
-        } else {
-          return views[name]
-            ? `import ${name} from '${getImportFileName(name, file)}'`
-            : viewNotFound(name)
-        }
+
+        return views[name]
+          ? `import ${name} from '${getImportFileName(name, file)}'`
+          : viewNotFound(name)
       }
     }
 
-    const data = {}
     const dependsOn = {}
     const dependedUpon = {}
```

Comment:
How about we rename `dependedUpon` to `responsibleFor` ?

Reply:
sounds good to me